FORMULA

```diff
@@ -0,0 +1,9 @@
+name: sudoers
+os: Debian, Ubuntu, Raspbian, RedHat, Fedora, CentOS, Suse, openSUSE
+os_family: Debian, RedHat, Suse
+version: 1.0.0rc
+release: 1
+minimum_version: 2016.11
+summary: Sudoers formula
+description: Formula to configure sudo
```

Comment:
`sudoers` perhaps?

Reply:
the program is named sudo :) I don't know why the formula is named that way

Reply:
The `README` doesn't help us here:

> Set up sudo and the sudoers included files.

So it does both!  Since there's confusion, maybe it's best to stick to `sudoers`, for consistency.

Reply:
except that "sudoers" does not mean anything, the program is sudo, the package is sudo, this is the name you find everywhere, see https://packages.debian.org/search?keywords=sudo&searchon=names&suite=stable&section=all for example.

Reply:
But the problem is, every reference _in the formula_ is for `sudoers`... the name of the formula, the directory structure, the pillar, etc.

Reply:
Or why don't we just write both, just like the `README`?  Meaning:

```suggestion
description: Formula to configure sudo and the sudoers included files
```

Or something like that.